### PR TITLE
add shifted softplus

### PR DESCRIPTION
**Pull request by mist714**
Add shifted-softplus for SchNet for stability
Add test of shifted softplus

## qm9 examples 

before

```
epoch       main/loss   main/mae    main/rmse   validation/main/loss  validation/main/mae  validation/main/rmse  elapsed_time
1           9.16072e+07  4635.03     5493.88     2.04751e+06           1172.89              1430.77               4.68606       
2           736497      626.273     809.699     175424                307.404              418.077               8.61846       
3           111058      246.223     331.111     83034.3               203.431              286.713               12.6977       
...
20          8439.32     56.8124     90.6312     9510.97               55.5326              92.7485               81.2714       
```

after
```
epoch       main/loss   main/mae    main/rmse   validation/main/loss  validation/main/mae  validation/main/rmse  elapsed_time
1           1665.32     28.0805     33.3155     163.623               7.94275              12.7757               4.86237       
2           106.288     4.85469     10.128      69.1701               3.48232              8.2747                9.02798       
3           59.911      2.95717     7.69376     41.7183               2.6097               6.42595               13.3118       
...
20          0.719011    0.659139    0.847248    0.741667              0.641484             0.85456               86.3188     
```

### CODE REVIEW

File: result/log [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
```
Comment (corochann) [inline result/log]: remove this

Comment (mottodora) [inline result/log]: Please delete this file.

File: chainer_chemistry/functions/activation/shifted_softplus.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import chainer
+from chainer import functions
+
+
+def shifted_softplus(x, beta=0.5):
```
Comment (corochann) [inline chainer_chemistry/functions/activation/shifted_softplus.py]: Can you use `beta` as the argument of `softplus` function and please introduce new value `shift` for shift purpose.
Please refer https://docs.chainer.org/en/stable/reference/generated/chainer.functions.softplus.html

Comment (corochann) [reply chainer_chemistry/functions/activation/shifted_softplus.py]: https://github.com/dmlc/dgl/blob/master/python/dgl/model_zoo/chem/layers.py#L128-L147

File: result/log (hunk 1 above)
Comment (mist714) [reply result/log]: done

**Comment by corochann** [comment]
SchNetReadout の部分も shifted_softplus に変えてもらって、性能変わるか見ていただけないでしょうか。

File: chainer_chemistry/functions/activation/shifted_softplus.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        beta (float): Parameter :math:`\\beta`.
+        shift (float): Shift Parameter 
+        threshold (float): threshold to avoid overflow
+
+    Returns:
+        output (Variable): Output variable whose shape is same with `x`
+    """
+    xp = chainer.cuda.get_array_module(x)
+    if isinstance(x, chainer.Variable):
+        cond = x.array > 20
```
Comment (corochann) [inline chainer_chemistry/functions/activation/shifted_softplus.py]: please use threshold

**Pull request merged**